reactor-core/src/test/java/reactor/core/publisher/SinksTest.java

```diff
@@ -165,6 +165,39 @@ void noReplayBeforeFirstSubscriberIfEarlyError() {
 		}
 	}
 
+	@Nested
+	class MulticastReplayDuration {
+
+		final Duration            duration = Duration.ofMillis(1000);
+		final Sinks.Many<Integer> replaySink = Sinks.many().replay().limit(duration);
+		final Flux<Integer>       flux = replaySink.asFlux();
+		final int 				  event = 12;
+
+		@Test
+		void lateSubscriberReceivesEventInRetentionTime() {
+			replaySink.emitNext(event, FAIL_FAST);
```

Comment:
nitpick: I'd add a comment referencing the issue to avoid somebody later changing the retention duration or the `minusMillis(1)` instruction for example

Reply:
added comment